pulser/sequence.py

```diff
@@ -258,6 +383,16 @@ def target(self, qubits, channel):
                 f"{self._channels[channel].max_targets} qubits at a time"
             )
 
+        if not self._building:
+            for q in qs:
+                if q not in self._qids and not isinstance(q, Parametrized):
+                    raise ValueError("All non-variable qubits must belong to "
```

Comment:
There are variable qubits ?

Reply:
I was thinking you can use a `Variable`, it works.